messaging/tests/test_messaging.py

```diff
@@ -191,17 +191,17 @@ def test_recv_one(self):
   def test_recv_one_or_none(self):
```

Comment:
let's add an assert raises to handle the ZMQ case so we can get this merged

Reply:
hmm, im not sure if i get that, what do you mean? the easiest workaround to make it pass on ZMQ, that i can think of is to add `zmq_sleep` after send

Reply:
https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertRaises

Reply:
======================================================================
FAIL: test_recv_one_or_none (messaging.tests.test_messaging.TestMessaging)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/project/cereal/messaging/tests/test_messaging.py", line 205, in test_recv_one_or_none
    self.assertIsInstance(recvd, capnp._DynamicStructReader)
AssertionError: None is not an instance of <class 'capnp.lib.capnp._DynamicStructReader'>
```

Reply:
I just mean it should mark the ZMQ case as expected to fail, see https://docs.python.org/3/library/unittest.html#skipping-tests-and-expected-failures